### DISCUSSION TITLE
Folder tree view in Find All References

### DISCUSSION THREAD

**Discussion by opsidjflksdf** [microsoft/vscode-discussions#1184] [Extension Development QnA] [answered]
If you right-click on a symbol to go to its references, besides Go To References, there's the option Find All References, which opens the references in a tool window rather than in the quickView. That's very nice. I was wondering if there would be a way to have the results displayed in a tree (based on the folders), just like the search results for find-in-files have the option to be displayed. I don't see the little icons to switch from tree view to list view and I couldn't find a setting. Any idea where to start if I wanted to implement that in an extension? 
Thanks in advance!

**Comment by isidorn** [comment, marked as answer]
Currently an extension can not control the Find All References view in such a way.
Your extension could contribute another view, and then have actions to toggle between different display styles.
